CHANGELOG.md

```diff
@@ -46,6 +46,11 @@ by add ibc default gen state and min gas price specification of `1ubbn`
 * [#93](https://github.com/babylonlabs-io/babylon/pull/93) fix genesis epoch
   initialization.
 
+### Misc Improvements
```

Comment:
this should probably go to `## Unreleased` section, after release of `v0.10.2` be moved there

Reply:
my bad. Will backport the fix to main as well